### PR TITLE
Add tools for after an interview

### PR DESCRIPTION
**Pull request by zoefarrell**

### CODE REVIEW

**Comment by zoefarrell** [comment]
I'm thinking we introduce this before the interviews and have them read the article and reflect before they meet with their group in the afternoon. I think it's ideally best to reflect on this stuff when you didn't just just have an interview and are not in a heightened emotional state.

**Comment by zoefarrell** [comment]
> I'm thinking we introduce this before the interviews and have them read the article and reflect before they meet with their group in the afternoon. I think it's ideally best to reflect on this stuff when you didn't just just have an interview and are not in a heightened emotional state.

Actually, I think we should separate introducing vulnerability hangovers and soft landings from the interview/assessment altogether. 

Maybe we block out 30 minutes end of day Wednesday? I could pop in to run this conversation, or you could if you want to. We could move Intro to CS to next week if we feel like that doesn't leave enough project time Wednesday?

File: module6/lessons/Week1/ToolsForAfterAnInterview.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+1. How are you emotionally right now? (Check in for self-doubt, insecurity, fatigue, cravings, wanting to hide)
+2. If you are having any vulnerability hangover symptoms, what can you do to create a "soft landing" and take care of yourself? (Think about if there is anything your group or other support people in your life can help you with here. You don't have to build your soft landing alone.)
+
+Then, in your group, take turns sharing as much as you're comfortable. Everyone processes differently, so it's normal if one person prefers to share briefly and the next chooses to share more and do more external processing. That being said, I encourage you to push yourself in sharing. Many people find that tough emotions have less power when they are named and shared.
+
+If you're a listener, remember that your job is to listen and maybe share empathy. Hold off from jumping in with solutions or ideas unless you're explicitly asked for that.
+
+### Conclusion
+
+We will give space for groups to do Interview Recaps and Emotions Check SIns after each Friday interview. We encourage you to also find time to support each other after your real interviews.
```
Comment (memcmahon) [inline module6/lessons/Week1/ToolsForAfterAnInterview.md]: This is reading like 'check SINS' 😆 

Also, Friday should be Thursday

File: module6/lessons/Week1/ToolsForAfterAnInterview.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+What parts of this blog post resonate with you?
+
+How does a vulnerability hangover show up for you?
+
+What are some ways that you can nurse your vulnerability hangover so you can continue to be brave? What’s your "soft landing"?
+
+Are there ways that your group can support you in your "soft landing"? For example, maybe there is something they can remind you to do for yourself or a mantra they can remind you of.
+
+When we come back together after today's interviews, you will discuss these questions with your group and then I'll ask for volunteers to share in the full group. Then we will move into an Interview Recap and Emotional Check-In for today's interview.
```
Comment (memcmahon) [inline module6/lessons/Week1/ToolsForAfterAnInterview.md]: I think we could reframe this as 'We will have time for you to come together after each week's interview to discuss these questions'

Comment (memcmahon) [reply module6/lessons/Week1/ToolsForAfterAnInterview.md]: That way this session could be moved around :)

File: module6/lessons/Week1/ToolsForAfterAnInterview.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+Start by taking time to individually reflect on the following two questions.
+
+1. How are you emotionally right now? (Check in for self-doubt, insecurity, fatigue, cravings, wanting to hide)
+2. If you are having any vulnerability hangover symptoms, what can you do to create a "soft landing" and take care of yourself? (Think about if there is anything your group or other support people in your life can help you with here. You don't have to build your soft landing alone.)
+
+Then, in your group, take turns sharing as much as you're comfortable. Everyone processes differently, so it's normal if one person prefers to share briefly and the next chooses to share more and do more external processing. That being said, I encourage you to push yourself in sharing. Many people find that tough emotions have less power when they are named and shared.
+
+If you're a listener, remember that your job is to listen and maybe share empathy. Hold off from jumping in with solutions or ideas unless you're explicitly asked for that.
```
Comment (memcmahon) [inline module6/lessons/Week1/ToolsForAfterAnInterview.md:56]: Love this reminder!

**Review by memcmahon — APPROVED** [review]

**Comment by memcmahon** [comment]
I agree with introducing this before Thursday - that will give them time to digest and come into the interviews with these reflections in mind.

File: module6/lessons/Week1/ToolsForAfterAnInterview.md (hunk 1 above)
Comment (zoefarrell) [reply module6/lessons/Week1/ToolsForAfterAnInterview.md]: Good catches!

File: module6/lessons/Week1/ToolsForAfterAnInterview.md (hunk 2 above)
Comment (zoefarrell) [reply module6/lessons/Week1/ToolsForAfterAnInterview.md]: Good call to make this more flexible. I'm thinking that reading the blog, individual reflection, small group sharing on these questions, and then whole group sharing is the focus of tomorrow.

And then maybe briefly going over the structure for Interview Recaps and Emotions Check-Ins. But I'm planning to guide students through the recap and check ins with a lot of structure this first time. So giving time for thinking, then going to breakout rooms, then introing next section, time for thinking, breakouts.

**Pull request merged**